src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,20 @@
+.calendar {
+  display: flex;
+  margin: auto;
+  width: $calendar-width;
+  padding: $calendar-padding * 2 $calendar-padding;
+  gap: $gap;
+  flex-wrap: wrap;
```

Comment:
vertical centering isn't necessary
11. [STYLES] - Don't be greedy, add some top paddings, **so the top row don't go over top of the page on hover.**

Reply:
Yes. In case you need it at the top you may use some top padding, but if you need it in the middle it won't help... Anyway, it is a good practice to try to place it in the middle instead of just adding top padding))